tests/integration/test_notebooks_gpu.py

```diff
@@ -19,20 +19,76 @@ def test_gpu_vm():
 @pytest.mark.integration
 @pytest.mark.gpu
 @pytest.mark.parametrize(
-    "size, expected_values",
+    "size, epochs, expected_values",
     [
-        ("1m", {"map": 0.024821, "ndcg": 0.153396, "precision": 0.143046, "recall": 0.056590}),
+        ("1m", 10, {"map": 0.024821, "ndcg": 0.153396, "precision": 0.143046, "recall": 0.056590}),
+        #("10m", 5, {"map": 0.024821, "ndcg": 0.153396, "precision": 0.143046, "recall": 0.056590})# takes too long
     ],
 )
-def test_ncf_integration(notebooks, size, expected_values):
+def test_ncf_integration(notebooks, size, epochs, expected_values):
     notebook_path = notebooks["ncf"]
     pm.execute_notebook(
         notebook_path,
         OUTPUT_NOTEBOOK,
         kernel_name=KERNEL_NAME,
-        parameters=dict(TOP_K=10, MOVIELENS_DATA_SIZE=size),
+        parameters=dict(TOP_K=10, 
+                        MOVIELENS_DATA_SIZE=size,
+                        EPOCHS=epochs,
+                        BATCH_SIZE=512),
     )
     results = pm.read_notebook(OUTPUT_NOTEBOOK).dataframe.set_index("name")["value"]
 
     for key, value in expected_values.items():
         assert results[key] == pytest.approx(value, rel=TOL)
+
+
+@pytest.mark.integration
+@pytest.mark.gpu
+@pytest.mark.parametrize(
+    "size, epochs, expected_values",
+    [
+        ("100k", 50, {"map": 0.051398, "ndcg": 0.204228, "precision": 0.183987, "recall": 0.105546,
+                  "map2": 0.049723, "ndcg2": 0.201361, "precision2": 0.180276, "recall2": 0.103631}),
+    ],
+)
+@pytest.mark.skip(reason="as of now, it takes too long to do a integration test")
```

Comment:
Why? NCF deep dive took 3 mins to train with 100k.

Reply:
I´ve been making some tests, NCF  deep dive notebook with 100K, 50 epochs and BS of 512 takes around 20min, however the notebook executed through papermill takes much longer. See results here: https://github.com/Microsoft/Recommenders/issues/466. I think we should skip these tests until we find out what is happening in the mentioned issue